platform/ext/target/nordic_nrf/common/nrf5340/gcc/startup_nrf5340.c

```diff
@@ -25,6 +25,10 @@
 #include "hw_init.h"
 #include "startup.h"
 
+__NO_RETURN __attribute__((naked)) void default_tfm_IRQHandler(void) {
```

Comment:
Why not in the common startup.c file?

Reply:
Because it did not compile when I did so.

I tried different things but I was not able to figure out why I could not do this.